ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Abstract.hs

```diff
@@ -36,6 +39,26 @@ import           Ouroboros.Consensus.Ledger.Basics
 import           Ouroboros.Consensus.Ticked
 import           Ouroboros.Consensus.Util (repeatedly, repeatedlyM)
 
+-- | " Validated " transaction or block
+--
+-- The ledger defines how to validate transactions and blocks. It's possible the
+-- type before and and validation may be distinct (eg Alonzo transactions),
+-- which originally motivated this family.
+--
+-- We also gain the related benefit that certain interface functions, such as
+-- 'reapplyLedgerBlock' can have a more precise type now. TODO
+--
+-- Similarly, the Node-to-Client mini protocols can explicitly indicate that the
+-- client trusts the blocks from the local server, by sending 'Validated'
+-- blocks. TODO
+--
+-- Note that validation has different implications for a transaction than for a
+-- block. In particular, a validated transaction can be " reapplied " to
+-- different ledger states, whereas a validted block must only be " reapplied "
+-- to the exact same ledger state (eg as part of rebuilding from an on-disk
+-- ledger snapshot).
+data family Validated x :: Type
```

Comment:
This is the new concept, now represented in the types.

Reply:
I've added another paragraph to the comment. Is that enough?

Reply:
Yes, I think that is enough. Thanks! (Resolving)